src/Exceptions/HttpConflict.php

```diff
@@ -8,4 +8,3 @@ class HttpConflict extends HttpException {
     public const DESCRIPTION = "Indicates that the request could not be completed due to a conflict with the current state of the resource.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces